### DISCUSSION TITLE
程序出现未知错误

### DISCUSSION THREAD

**Discussion by hijijibao** [Meloong-Git/PCL#2302] [提问] [answered]
程序出现未知错误：“System.windows.Media.Fonts”的类型初始值设定引发异常。
-> No FontFamily element fiund in FontFanilyCollection that mataches current OS or greater: Windows7SP1
在 PCL.ModSecret.SecretOnApplicaticationStart()
在 PCL.Application.Application_Startup(Object sender,StartuoEventArgs e)
错误类型：System.IO.FileFormatException

遇到这个怎么解决
大佬帮帮我唉！！！！

**Comment by zkitefly** [comment, marked as answer]
按照教程操作：https://support.microsoft.com/zh-cn/topic/f941cb72-016c-4207-f570-06c90c8a7559#:~:text=%E6%96%B9%E6%B3%95%201

> 方法来自：https://github.com/JosefNemec/Playnite/issues/422
